packages/forklift-console-plugin/src/modules/Plans/views/create/components/SearchInputProvider.tsx

```diff
@@ -23,12 +23,19 @@ export const SearchInputProvider: React.FunctionComponent<SearchInputProviderPro
     filterDispatch({ type: 'SET_NAME_FILTER', payload: value });
   };
 
+  const onChange: (event: React.FormEvent<HTMLInputElement>, value: string) => void = (
+    event,
```

Comment:
```suggestion
    _event,
```

do you we use the `event` value ?